src/Nox.Types.EntityFramework/Types/HashedText/HashedTextDatabaseConfigurator.cs

```diff
@@ -13,11 +11,26 @@ public class HashedTextDatabaseConfigurator : INoxTypeDatabaseConfigurator
 
     public bool IsDefault => true;
 
-    public void ConfigureEntityProperty(NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, EntityTypeBuilder builder, NoxSimpleTypeDefinition property, Entity entity, bool isKey)
+    public void ConfigureEntityProperty(
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState,
+        IEntityBuilderAdapter builder,
+        NoxSimpleTypeDefinition property,
+        Entity entity,
+        bool isKey)
     {
-        builder
-            .OwnsOne(typeof(HashedText), property.Name)
-            .Ignore(nameof(HashedText.Value));
+        var ownedNavigation = builder
+            .OwnsOne(typeof(HashedText), property.Name);
+
+        if (ownedNavigation is EntityTypeBuilder etb)
```

Comment:
why this complexity, this should be hidden behind the adapter?

Reply:
Let's discuss this